testslide/__init__.py

```diff
@@ -20,24 +20,39 @@
 import unittest
 import warnings
 from contextlib import contextmanager
-from typing import List
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterator,
+    List,
+    Optional,
+    TextIO,
+    Tuple,
+    Type,
+    Union,
+)
 
 import testslide.matchers
 import testslide.mock_callable
 import testslide.mock_constructor
 import testslide.patch_attribute
 from testslide.strict_mock import StrictMock  # noqa
 
+if False:
+    # hack for Mypy
```

Comment:
Why? If this is some chicken / egg issue with imports, quoting the types usually helps.

Reply:
if i just quote the type, then mypy will fail with cannot find type "asdf" if i dont "import" the depending module like so. also i can't properly import that due to circular depencendy things. :D

Reply:
will give typing.TYPE_CHECKING a try

Reply:
yep, thx